src/igvfd/schemas/changelogs/tabular_file.md

```diff
@@ -2,6 +2,15 @@
 
 ### Minor changes since schema version 13
 
+* Extend `transcriptome_annotation` enum list to include `GENCODE 28`.
+* Extend `transcriptome_annotation` enum list to include `GENCODE 25`.
+* Extend `transcriptome_annotation` enum list to include `GENCODE M17`.
+* Extend `transcriptome_annotation` enum list to include `GENCODE 28, GENCODE M17`.
+* Extend `assembly` enum list to include `hg19`.
+* Extend `assembly` enum list to include `Cast - GRCm39`.
```

Comment:
I don't think this was newly added, it just got moved lower in the list, so we don't need to include a new changelog entry for it

Reply:
done.